File: Learning Core Data/SaveDemo/SaveDemo/ViewController.m
//
//  ViewController.m
//  SaveDemo
//
//  Created by Wes Van der Klomp on 4/21/16.
//  Copyright © 2016 wvdk. All rights reserved.
//

#import "ViewController.h"
#import "AppDelegate.h"

@implementation ViewController {
     NSManagedObjectContext *moc;
}

- (IBAction)createObject:(id)sender {
    
    
    NSManagedObject *myMO = [NSEntityDescription insertNewObjectForEntityForName:@"Course" inManagedObjectContext:moc];
    
    [myMO setValue:@"Core Data 101" forKey:@"Title"];
    [myMO setValue:@"Wes" forKey:@"Author"];
    [myMO setValue:[NSDate date] forKey:@"releaseDate"];
    
//    NSError *error = nil;
    
//    if () {
//        NSLog(@"Ahhh! %@", error);
//    }

//    [moc save];
    
    NSError *error = nil;
    if (![moc save:&error])
    {
                NSLog(@"Ahhh! %@", error);
    }
}

- (void)viewDidLoad {
    [super viewDidLoad];

    moc = [(AppDelegate *)[[NSApplication sharedApplication] delegate] managedObjectContext];
    

    
    // Do any additional setup after loading the view.
}

- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}

@end

File: PeriodicalScreenCap/PeriodicalScreenCap/ViewController.swift
//
//  ViewController.swift
//  PeriodicalScreenCap
//
//  Created by Wesley Van der Klomp on 11/17/16.
//  Copyright © 2016 Wesley Van der Klomp. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {

    @IBOutlet weak var pathField: NSTextField!
    @IBOutlet weak var timeIntervalFields: NSTextField!
    
    let task = Process()
    
    
    @IBAction func start(_ sender: NSButton) {
        print("Starting")
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

